### ISSUE TITLE
Proton8.0 does not show up on Steam Deck 

### ISSUE THREAD

**Issue by dtantono** [ValveSoftware/steam-for-linux#9353]
Hi

I install the proton 8.0 (the directory is /home/deck/.local/share/Steam/steamapps/common/Proton 8.0/) by Desktop mode > Steam > In Library select "All" and "Tools" > Search for Proton 8.0 > Install. and it didn't show up on Steam (Setting -> Steam Play). even after reboot. what do you think?
![image](https://user-images.githubusercontent.com/29157187/232926786-c6a162d5-66d6-4b40-a7ad-9665dffc2a22.png)

PS: it shows up on Heroic BTW

**Comment by kisak-valve** [comment]
Hello @dtantono, managing select-able Steam Play compatibility tools is handled by the Steam client, not Proton itself, so I've transferred this issue report.

**Comment by dtantono** [comment]
@kisak-valve yeah, my Ubuntu PC can detect newest proton 8.0. But, SteamOS cannot which is weird.

**Comment by CB9TOIIIA** [comment]
I have the same problem. Not display in list view steam os proton 8.
